src/postgres/backend/access/common/printtup.cpp

```diff
@@ -122,7 +123,8 @@ SetRemoteDestReceiverParams(DestReceiver *self, Portal portal)
 }
 
 static void
-printtup_startup(DestReceiver *self, int operation, TupleDesc typeinfo)
+printtup_startup(DestReceiver *self, int operation, TupleDesc typeinfo,
+								 MemcachedState *mc_state)
 {
 	DR_printtup *myState = (DR_printtup *) self;
 	Portal		portal = myState->portal;
```

Comment:
This file is using tabs. It's better to replace them with spaces according to the style guide.